examples/spark/images/base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:latest
+FROM java:openjdk-8-jdk
```

Comment:
simple scan of the zeppelin docs show java 1.7 - https://zeppelin.incubator.apache.org/docs/install/install.html

maybe that means 1.7 only or 1.7+, pinning on java 8 start makes sense